internal/core/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+add_library(vdscore
+  attribute.cpp
+  axis.cpp
+  boundingbox.cpp
+  cppapi_data.cpp
+  cppapi_metadata.cpp
+  datahandle.hpp
+  datahandle.cpp
+  direction.cpp
+  metadatahandle.cpp
+  regularsurface.cpp
+  subvolume.cpp
+  verticalwindow.cpp
+)
+
+find_package(openvds CONFIG REQUIRED)
+
+target_include_directories(vdscore
+  PUBLIC openvds::openvds
+)
```

Comment:
But we to add the headers to the target: 
```
target_include_directories(vdscore
    PUBLIC ${CMAKE_SOURCE_DIR}/internal/core>
)
```

Which means we don't need to do that in `gtest/CMakeLists.txt`

(If you want to do it _right_ you can also make 2 lists of headers, public and private, before adding them in `target_include_directories`. But as this is purely for testing, it might be nice that all headers are public to the test suite)